dataprofiler/tests/test_dp_logging.py

```diff
@@ -0,0 +1,71 @@
+import unittest
+import logging
+
+from dataprofiler import DataLabeler, dp_logging
+from dataprofiler.labelers.regex_model import RegexModel
+
+
+class TestDPLogging(unittest.TestCase):
+
```

Comment:
we should add a test that asserts the default is set to `INFO`.

Reply:
Added